mobile/src/home-view.js

```diff
@@ -36,7 +36,9 @@ class HomeView extends Component {
       marginTop: 18, 
       animation: "none",
       title: "Q & A",
-      questionAsk: false
+      questionAsk: false,
+      questionError: "Ask Question",
+      topBorder: "#EFEFEF"
```

Comment:
This doesn't appear to be used

Reply:
This is only passed down to the cell to show errors